.cirrus.yml

```diff
@@ -1,18 +1,49 @@
+env:
+  HOME: /root
+  # GOPATH is per-user. Here we run as root, so we put it in the root home directory.
+  GOPATH: ${HOME}/go
+  PATH: /usr/local/go/bin:${GOPATH}/bin:${PATH}
+  #
+  GO_PKG: go1.24.5.linux-amd64.tar.gz
+  TASKFILE_VERSION: v3.44.0
+
 task:
-  container:
-    image: golang:latest
-    # https://cirrus-ci.org/guide/linux/#kvm-enabled-privileged-containers
-    # kvm: true
+  # https://cirrus-ci.org/guide/custom-vms/
+  # Cirrus CI uses Google Cloud Platform for running all instances, except macos_instance.
+  # Things like Docker Builder and freebsd_instance are syntactic sugar for launching
+  # Compute Engine instances from a particular set of images.
+  # With compute_engine_instance it is possible to use any publicly available image for
+  # running your Cirrus tasks in.
+  #
+  # List all the public images of a given "project":
+  # gcloud compute images list --project freebsd-org-cloud-dev --no-standard-images
+  #
+  compute_engine_instance:
+    image_project: cirrus-images # Name of the Cirrus CI public GCP project.
+    image: family/docker-kvm
+    platform: linux
+    # architecture: arm64
+    architecture: amd64
+    cpu: 4
```

Comment:
they offer 4vCPU and 16GB instances as part of a free account? They are quite generous...